From: Alexander Aring <aar@pengutronix.de>
Date: Tue, 26 Apr 2016 11:25:24 +0200
Subject: [PATCH] main: interface needs argument

The interface argument needs an argument otherwise optarg is NULL.

Signed-off-by: Alexander Aring <aar@pengutronix.de>
---
 programs/sunshine/main.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/programs/sunshine/main.cpp b/programs/sunshine/main.cpp
index 2128eb616fac..b005ccf42fd9 100644
--- a/programs/sunshine/main.cpp
+++ b/programs/sunshine/main.cpp
@@ -34,7 +34,7 @@ char *progname;
 static struct option const longopts[] =
 {
     { "help",      0, 0, '?'},
-    { "interface", 0, 0, 'i'},
+    { "interface", 1, 0, 'i'},
     { "daemon",    0, 0, 'D'},
     { "prefix",    1, NULL, 'p'},
     { "ignore-pio",0, NULL, 'P'},
